infra/cifuzz/actions/build_fuzzers/build_fuzzers_entrypoint.py

```diff
@@ -81,7 +81,7 @@ def main():
   out_dir = os.path.join(workspace, 'out')
   if cifuzz.check_fuzzer_build(out_dir):
     return 0
-  return 1
+  return error_code
```

Comment:
"error_code" should probably be called "returncode"
(You can use an underscore, subprocess doesn't though).